lib/google-auth.js

```diff
@@ -0,0 +1,33 @@
+import google from 'googleapis';
+import util from './util';
+
+let KEY;
+util.readJsonFromFile('./google-service-api-json-key.json', data => {
+    KEY = data;
+});
+
+class GoogleAuth{
+    constructor() {
+        return this;
+    }
+
+    authenticate(callback){
+        let jwtClient = new google.auth.JWT(
```

Comment:
Prefer using `const` instead `let` whenever possible. See this part of the nice AirBnB guide here: https://github.com/airbnb/javascript#references